docs/content/releases.md

```diff
@@ -6,6 +6,38 @@ doctypes:
 title: Releases
 toc: true
 weight: 2100
+## 3.6.1
+
+02 Jul 2024
+
+### <i class="fa-solid fa-bomb"></i> Breaking Changes
+
+### <i class="fa-solid fa-rocket"></i> Features
+
+### <i class="fa-solid fa-bug-slash"></i> Fixes
+
+### <i class="fa-solid fa-box"></i> Helm Chart
+
+### <i class="fa-solid fa-upload"></i> Dependencies
+
+### <i class="fa-solid fa-download"></i> Upgrade
+
```

Comment:
The date here needs to be changed to July 3rd.

We can also remove the Breaking Change field since we only use that if we're doing a major version bump.
Other than that, we need to add the [PRs that were merged with the 3.6.1 Milestones](https://github.com/nginxinc/kubernetes-ingress/milestone/25?closed=1)